src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -130,7 +130,12 @@ class Dummy : IDisposable
 
         public bool ReportingAnalytics
         {
-            get { return preferences != null && preferences.IsAnalyticsReportingApproved; }
+            get
+            {
+                return preferences != null 
```

Comment:
@QilongTang should we also do this for ReportingUsage right below?

Reply:
@mjkkirschner I think we do, I haven't found a way to trigger log Pii, are you aware of any?